message-index/messages/GHC-68686/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Cannot parse LANGUAGE pragma
+summary: The arguments to the LANGUAGE pragma could not be parsed
+severity: error
+introduced: 9.6.1
```

Comment:
Please add needed extensions.

Reply:
I don't think there are any...

Reply:
Indeed, this is for all language pragmas